### ISSUE TITLE
docker install fails

### ISSUE THREAD

**Issue by danielslyman** [crater-invoice/crater#1080]
Hey guys,

so I followed the docker installation instructions 1:1 multiple times and always get to the same point after executing:

./docker-compose/setup.sh

from the root folder of the cloned repo.

```
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - psr/log is locked to version 2.0.0 and an update of this package was not requested.
    - psr/log 2.0.0 requires php >=8.0.0 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 2
    - ramsey/uuid is locked to version 4.3.1 and an update of this package was not requested.
    - ramsey/uuid 4.3.1 requires php ^8.0 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 3
    - symfony/css-selector is locked to version v6.0.3 and an update of this package was not requested.
    - symfony/css-selector v6.0.3 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 4
    - symfony/deprecation-contracts is locked to version v3.0.1 and an update of this package was not requested.
    - symfony/deprecation-contracts v3.0.1 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 5
    - symfony/event-dispatcher is locked to version v6.0.9 and an update of this package was not requested.
    - symfony/event-dispatcher v6.0.9 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 6
    - symfony/event-dispatcher-contracts is locked to version v3.0.1 and an update of this package was not requested.
    - symfony/event-dispatcher-contracts v3.0.1 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 7
    - symfony/string is locked to version v6.0.9 and an update of this package was not requested.
    - symfony/string v6.0.9 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 8
    - symfony/translation is locked to version v6.0.9 and an update of this package was not requested.
    - symfony/translation v6.0.9 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 9
    - symfony/translation-contracts is locked to version v3.0.1 and an update of this package was not requested.
    - symfony/translation-contracts v3.0.1 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 10
    - psr/cache is locked to version 3.0.0 and an update of this package was not requested.
    - psr/cache 3.0.0 requires php >=8.0.0 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 11
    - symfony/filesystem is locked to version v6.0.9 and an update of this package was not requested.
    - symfony/filesystem v6.0.9 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 12
    - symfony/options-resolver is locked to version v6.0.3 and an update of this package was not requested.
    - symfony/options-resolver v6.0.3 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 13
    - symfony/stopwatch is locked to version v6.0.5 and an update of this package was not requested.
    - symfony/stopwatch v6.0.5 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
  Problem 14
    - symfony/deprecation-contracts v3.0.1 requires php >=8.0.2 -> your php version (7.4.32) does not satisfy that requirement.
    - fakerphp/faker v1.19.0 requires symfony/deprecation-contracts ^2.2 || ^3.0 -> satisfiable by symfony/deprecation-contracts[v3.0.1].
    - fakerphp/faker is locked to version v1.19.0 and an update of this package was not requested.


Warning: require(/var/www/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/artisan on line 18

Fatal error: require(): Failed opening required '/var/www/vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /var/www/artisan on line 18

Warning: require(/var/www/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/artisan on line 18

Fatal error: require(): Failed opening required '/var/www/vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /var/www/artisan on line 18
```

Any idea how I can remedy the situation?

Thanks,
Daniel

**Comment by dionikink** [comment]
I had the same issue just now. It seems to be related to the PHP version. There's an open pull request that updates the PHP version in the Dockerfile. I applied it manually and that got me past the error.

See here: https://github.com/crater-invoice/crater/pull/1078/commits/5a7514f8a1802f35c1b483ed5ad47f1a5905376d

**Comment by mohitpanjwani** [comment]
Yes, it seems that the updated composer lock file caused an issue with the docker setup. 

I've merged the PR to use PHP 8.1 image instead of 7.4 which is working perfectly.

**Comment by danielslyman** [comment]
Thanks!
